boards/px4/fmu-v5x/src/board_config.h

```diff
@@ -186,18 +186,20 @@
 #define GPIO_SPI3_nCS2_BMI088_GYRO  /* PI8 */  (GPIO_OUTPUT|GPIO_PUSHPULL|GPIO_SPEED_2MHz|GPIO_OUTPUT_SET|GPIO_PORTI|GPIO_PIN8)
 
 /*  Define the SPI3 Data Ready interrupts */
+#define GPIO_DRDY_BMI088_INT1_ACCEL /* PI6  */  (GPIO_INPUT|GPIO_FLOAT|GPIO_EXTI|GPIO_PORTI|GPIO_PIN6)
+//#define GPIO_DRDY_BMI088_ACCEL_INT2 /* PF2 */  (GPIO_OUTPUT|GPIO_PUSHPULL|GPIO_SPEED_2MHz|GPIO_OUTPUT_SET|GPIO_PORTF|GPIO_PIN2)
+#define GPIO_DRDY_BMI088_INT3_GYRO  /* PI7  */  (GPIO_INPUT|GPIO_FLOAT|GPIO_EXTI|GPIO_PORTI|GPIO_PIN7)
+// #define GPIO_DRDY_BMI088_GYRO_INT4  /* PF4 */  (GPIO_OUTPUT|GPIO_PUSHPULL|GPIO_SPEED_2MHz|GPIO_OUTPUT_SET|GPIO_PORTF|GPIO_PIN4)
 
-#define GPIO_SPI3_DRDY1_BMI088_INT1_ACCEL /* PI6  */  (GPIO_INPUT|GPIO_FLOAT|GPIO_EXTI|GPIO_PORTI|GPIO_PIN6)
-#define GPIO_SPI3_DRDY2_BMI088_INT3_GYRO  /* PI7  */  (GPIO_INPUT|GPIO_FLOAT|GPIO_EXTI|GPIO_PORTI|GPIO_PIN7)
 
 /*  SPI3 off */
 
 #define GPIO_SPI3_SCK_OFF   _PIN_OFF(GPIO_SPI3_SCK)
 #define GPIO_SPI3_MISO_OFF  _PIN_OFF(GPIO_SPI3_MISO)
 #define GPIO_SPI3_MOSI_OFF  _PIN_OFF(GPIO_SPI3_MOSI)
 
-#define GPIO_DRDY_OFF_SPI3_DRDY1_BMI088    _PIN_OFF(GPIO_SPI3_DRDY1_BMI088_INT1_ACCEL)
-#define GPIO_DRDY_OFF_SPI3_DRDY2_BMI088    _PIN_OFF(GPIO_SPI3_DRDY2_BMI088_INT3_GYRO)
+#define GPIO_DRDY_OFF_BMI088_INT1_ACCEL    _PIN_OFF(GPIO_DRDY_BMI088_INT1_ACCEL)
```

Comment:
Changing the name removed the power domain information from the pins. - I would not do this.